docs/installation.rst

```diff
@@ -76,3 +76,48 @@ Please add the following to your ``settings.py``:
 
 Finally let your OpenID connect OP know about your callback URL. In our example this is:
 ``http://127.0.0.1:8000/oidc/callback/``.
+
+
+Additional optional configuration
+=================================
+
+Connecting OIDC user identities to Django users
+-----------------------------------------------
+
+By default, mozilla-django-oidc looks up a Django user matching the email field
+to the email address returned in the user info data from the OIDC provider.
+
+This means that no two users in the Django user table can have the same email
+address. Since the email field is not unique, it's possible that this can
+happen. Especially if you allow users to change their email address. If it ever
+happens, then the users in question won't be able to authenticate.
```

Comment:
Does it make sens to add here a couple of lines about the `default_username_algo` and how it is used to populate the django username?